### PR TITLE
test: add e2e orch test

### PR DESCRIPTION
**Pull request by SweeXordious**
## Overview

Closes https://github.com/celestiaorg/orchestrator-relayer/issues/165

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: e2e/orchestrator_test.go [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+
+	dcConfirm, err := network.GetDataCommitmentConfirmByHeight(ctx, dht, network.DataCommitmentWindow-2, CORE0EVMADDRESS)
+	// assert the confirm exist
+	require.NoError(t, err)
+	require.NotNil(t, dcConfirm)
+	// assert that it carries the right evm address
+	assert.Equal(t, CORE0EVMADDRESS, dcConfirm.EthAddress)
+}
+
+func TestOrchestratorWithTwoValidators(t *testing.T) {
```
Comment (evan-forbes) [inline e2e/orchestrator_test.go:66]: a quick doc to explain what is being tested here would be nice, but I'm not blocking on it

File: e2e/orchestrator_test.go [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+	"time"
+
+	"github.com/celestiaorg/orchestrator-relayer/cmd/qgb/helpers"
+	qgbtesting "github.com/celestiaorg/orchestrator-relayer/testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestOrchestratorWithOneValidator(t *testing.T) {
```
Comment (evan-forbes) [inline e2e/orchestrator_test.go:16]: same comment as below

**Review by evan-forbes — APPROVED** [review]
nice! this is cool

File: e2e/orchestrator_test.go (hunk 1 above)
Comment (SweeXordious) [reply e2e/orchestrator_test.go:66]: https://github.com/celestiaorg/orchestrator-relayer/issues/205

File: e2e/orchestrator_test.go (hunk 2 above)
Comment (SweeXordious) [reply e2e/orchestrator_test.go:16]: https://github.com/celestiaorg/orchestrator-relayer/issues/205

**Pull request merged**